Parse/src/test/java/com/parse/ParseUserTest.java

```diff
@@ -699,13 +699,13 @@ public void testlinkWithInBackgroundWithSaveAsyncFailure() throws Exception {
         partialMockUser.linkWithInBackground(authType, authData);
     linkTask.waitForCompletion();
 
-    // Make sure new authData is added
-    assertSame(authData, partialMockUser.getAuthData().get(authType));
```

Comment:
This is the exact same test/comment from `testlinkWithInBackgroundWithSaveAsyncSuccess()` above, which makes me think it was expected in the test only because it was copied